---
layout: about
title: about
permalink: /
subtitle: Bioinformatics ðŸ§¬ Cloud computing ðŸ§¬ FAIR Data

profile:
  align: right
  image: ke-profile.jpeg
  image_circular: false # crops the image to make it circular
  address: >
    <p>Burlington, Vermont</p>

news: false  # includes a list of news items
latest_posts: false  # includes a list of the newest posts
selected_papers: true # includes a list of papers marked as "selected={true}"
social: true  # includes social icons at the bottom of the page
---

I am currently a Bioinformatics Analyst at Genentech in the Early Research & Development arm. My work is focused on improving data utility and cloud computing, providing bioinformatics support to scientists and analysts across clinical trials and multi-modal data types.

Primarily focused on whole exome sequencing and ctDNA/cfDNA assays, I develop workflows (Nextflow/Snakemake/WDL) that provide analysis-ready objects to stakeholders while maintaining end-to-end provenance of metadata. Currently working towards the migration of HPC & slurm managed workflows to AWS, focusing on efficiency and cost without sacrificing data quality and accessibility.

My graduate research focused on applying omics assays to issues within the field of animal & farm health, such as bovine vaccine design and antimicrobial resistance on poultry farms. Through a One Health and ecological perspective, I worked to leverage high throughput techniques such as shotgun metagenomics to address gaps within our agricultural system and provide feedback for ongoing management practices.

During my time working for the Vermont Integrative Genomics Resource Bioinformatics core, I worked on a wide array of NGS workflows and applications. These included metagenomics, single-cell RNA sequencing, assembly of novel genomes, RNAseq and more. From vaccine trials to field research, this core facility provided support to projects across Vermont and other regions. See more on my [publications page](https://keckstrom.github.io/publications/). I was also a co-instructor for MMG231 (Bioinformatics & Data Analysis) & MMG232 (Advanced Bioinformatics), focusing on the fundamentals of HPC computing, basic Unix, and metagenomics.

I am passionate about open access software and data reproducibility, especially data [FAIR](https://www.nature.com/articles/sdata201618)-ness from experimental design to final publication. Initially starting as a bench scientist, I am skilled in both computational and wet lab procedures, bridging the gap between the design and execution of complex NGS experiments with a clear understanding of the complexities of both settings. By providing metadata rich and user-friendly deliverables, data reuse and reproducibility are improved and the data lifecycle is maintained from start to finish. 
